pkg/status/registry.go

```diff
@@ -21,10 +25,17 @@ type Registry interface {
 	Key() string
 	HasError() bool
 	GetLogger() *log.Logger
+	GetAlerter() alerts.Alerter
 }
 
 var _ Registry = &registry{}
 
+const (
+	alertChannel = "status_registry"
```

Comment:
@tanveergill I hardcoded this because status registry is for our internal use
We do not configure alert channels in alerter itself (it's not the same as alerter component) - https://github.com/fluxninja/aperture/blob/main/pkg/alerts/provide.go#L15
I can change `GetAlerter` method to also specify the channel list if you want